client/src/components/top-bar/index.tsx

```diff
@@ -7,7 +7,7 @@ export default (props: any) => {
   return (
     <>
       <style>{getStyle(theme)}</style>
-      <Menu selectable={false} theme="dark" mode="horizontal">
+      <Menu className="menu" selectable={false} theme="dark" mode="horizontal">
```

Comment:
@JithinKS97 shouldn't theme be dynamic?

Reply:
Are you refering to the className that I added? I added that for providing a drop shadow.

Reply:
@JithinKS97 no, theme='dark'..shouldn't that be theme={theme}

Reply:
Theme="dark" is a prop provided by ant-design library. I think we can we can revisit this section later if we are planning to introduce another theme.

Reply:
okay